### PR TITLE
[IP-1] Expenses Module

### PR DESCRIPTION
**Pull request by Aeolun**
_Sorry, messed up my initial pull request. Github is (more or less) new to me._

Not sure you want this in v1, but well, I needed it, so it's going to be in my branch at least :smiley: 

From what I gather the intended v2 expenses module would be integrated with products, but that was beyond the scope of my need (for now).

In the end I'd like the system to be able to handle recurring expenses, so I'll work on this a bit more later. Please let me know what you think.

### CODE REVIEW

**Comment by Kovah** [comment]
Hey! Thanks for the submission!
I will review this later, in the meantime, could you change the branch to merge this to `v1.5.0`? I added the branch, it's even with 1.4.7.

File: index.php [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
 		case 'testing':
 		case 'production':
-			error_reporting(0);
```
Comment (Kovah) [inline index.php]: Please revert this a error reporting should be 0 for normal apps in production.

**Comment by justjotme** [comment]
Is there a missing tax_rate_id in the ip_expenses table?

I had a database error then I added tax_rate_id in the ip_expenses table and it worked.

**Comment by johnfoley** [comment]
Thank you for your mail.

Please note: due to bereavement the studio is closed on Monday 1 August and Tuesday 2 August 2016.

I will respond to your mail when we reopen on Wednesday 3 August 2016.

If your enquiry is urgent please do not hesitate to contact me on mobile

File: index.php (hunk 1 above)
Comment (Kovah) [inline index.php]: @Aeolun

**Comment by Smuuch** [comment]
Hi guys, is there anywhere we can buy the completed expense module for invoiceplane?

**Comment by chrispytorius** [comment]
I finished this module

you can get it here:
https://github.com/InvoicePlane/InvoicePlane/pull/688

**Pull request closed without merging**